### PR TITLE
wording change on FAQ page

### PR DESCRIPTION
**Pull request by thea-exe**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [ X] Refactoring

### What's new?

- [ ] PR closes #<issue_number>
- [ X] <I went to [FAQ](https://kodadot.xyz/rmrk/faq) and changed "Frequently Asked Question" to "Frequently Asked Questions and changed "NO TIME FOR READING?" to "Don't have time to read?">

### Before submitting Pull Request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [X] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [X] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-04-20 at 1 00 02 AM

### CODE REVIEW

**Review by roiLeo — CHANGES_REQUESTED** [review]
- check `langDir/en.json` file and you will find the translation (line :77)
- it would be nice to add the last two lines in translation file

**Comment by thea-exe** [comment]
@roiLeo I understand that it is in the en.json folder, however, "Frequently Asked Question" isn't proper, especially referring to multiple questions.

I'll make sure to update the en.json file with the last two lines though

**Review by roiLeo — COMMENTED** [review]
this change will break other "Frequently Asked Question" translation
here's what i would do:
- change json file like this:
```json
"faq": {
    "title": "Frequently Asked Questions",
    "box_heading1": "Don't have time to read?",
    "box_heading2": "No problem, check out",
```
- create in all other json files the `faq.title` & copy "Frequently Asked Question" line into this one
- then change in `components/common/FrequentlyAsked.vue` {{ $t('faq.title') }}

**Comment by thea-exe** [comment]
Oh! I understand now— let me implement this in the morning :)

**Comment by thea-exe** [comment]
I decided to add 
`"box_heading1": "",
    "box_heading2": "",`
just in case people wanted to translate those lines!

File: components/common/FrequentlyAsked.vue [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
       class="text-bold text__stroked heading heading-is-6 is-size-1-mobile mb-6">
-      {{ $t('Frequently Asked Question') }}
+      {{ $t('faq.title') }}
     </h1>
 
     <p
       class="faq__box heading is-size-4-mobile is-size-3-tablet text-bold is-flex-inline mb-6">
-      No time for reading? <br />
-      No problem. Check out <br />
+      Don't have time to read? <br />
```
Comment (roiLeo) [inline components/common/FrequentlyAsked.vue:10]:
```suggestion
      {{ $t('faq.box_heading1') }} <br />
```

File: components/common/FrequentlyAsked.vue [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
+      No problem, check out <br />
```
Comment (roiLeo) [inline components/common/FrequentlyAsked.vue:11]:
```suggestion
      {{ $t('faq.box_heading2') }}<br />
```

File: langDir/bn.json [hunk 3]
```diff
@@ -57,6 +57,9 @@
   "Read our story, how we started.": "আমাদের গল্পটি পড়ুন, আমরা কীভাবে শুরু করেছি।",
   "Frequently Asked Question": "সচরাচর জিজ্ঞাস্য(এফ.এ.কিউ)।",
   "faq": {
+    "title": "সচরাচর জিজ্ঞাস্য(এফ.এ.কিউ)।",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/bn.json:62]:
```suggestion
```

File: langDir/de.json [hunk 4]
```diff
@@ -65,8 +65,10 @@
   "Layout Options": "Anzeige-Optionen",
   "Gallery Items Per Page": "Gallerie-Einträge pro Seite",
   "Collections Per Page": "Sammlungen pro Seite",
-  "Frequently Asked Question": "Frequently Asked Question",
   "faq": {
+    "title": "Frequently Asked Questions",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/de.json:71]:
```suggestion
```

File: langDir/es.json [hunk 5]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "¿Eres nuevo? Consulta nuestras Preguntas Frecuentes",
   "Feedback? We\\'d be happy to discuss it.": "¿Tienes algún comentario? Lo discutiremos contigo encantados.",
   "Read our story, how we started.": "Lee nuestra historia, nuestros inicios.",
-  "Frequently Asked Question": "Preguntas Frecuentes",
   "faq": {
+    "title": "Preguntas Frecuentes",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/es.json:61]:
```suggestion
```

File: langDir/fr.json [hunk 6]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "Vous êtes nouveau ici ? Nous avons une F.A.Q.",
   "Feedback? We\\'d be happy to discuss it.": "Des commentaires? Nous serions heureux d'en discuter.",
   "Read our story, how we started.": "Lisez notre histoire, comment nous avons commencé.",
-  "Frequently Asked Question": "Questions fréquentes",
   "faq": {
+    "title": "Questions fréquentes",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/fr.json:61]:
```suggestion
```

File: langDir/hi.json [hunk 7]
```diff
@@ -57,6 +57,9 @@
   "Read our story, how we started.": "Read our story, how we started.",
   "Frequently Asked Question": "Frequently Asked Question",
   "faq": {
+    "title": "Frequently Asked Questions",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/hi.json:62]:
```suggestion
```

File: langDir/id.json [hunk 8]
```diff
@@ -74,10 +74,12 @@
   "Layout Options": "Pengaturan Tataletak",
   "Gallery Items Per Page": "Item Galeri Per Halaman",
   "Collections Per Page": "Koleksi Per Halaman",
-  "Frequently Asked Question": "Pertanyaan yang sering di ajukan",
   "I am helping to cover costs": "Saya membantu menutupi biaya",
   "I do not want to support": "Saya tidak ingin mendukung",
   "faq": {
+    "title": "Pertanyaan yang sering di ajukan",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/id.json:82]:
```suggestion
```

File: langDir/it.json [hunk 9]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "Sei nuovo qui? Abbiamo una F.A.Q.",
   "Feedback? We\\'d be happy to discuss it.": "Feedback? Saremo felici di discuterne.",
   "Read our story, how we started.": "Leggi la nostra storia, come tutto è cominciato.",
-  "Frequently Asked Question": "Domande frequenti",
   "faq": {
+    "title": "Domande frequenti",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/it.json:61]:
```suggestion
```

File: langDir/jp.json [hunk 10]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "初めての方はFAQを見てみましょう。",
   "Feedback? We\\'d be happy to discuss it.": "フィードバックがありますか？ディスカッション大歓迎です。",
   "Read our story, how we started.": "このプロジェクトをスタートした私たちのストーリーはこちら。",
-  "Frequently Asked Question": "FAQ（良くある質問）",
   "faq": {
+    "title": "FAQ（良くある質問）",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/jp.json:61]:
```suggestion
```

File: langDir/ko.json [hunk 11]
```diff
@@ -55,9 +55,11 @@
   "New here? We have F.A.Q.": "새로왔는가? 질의응답 코너이다.",
   "Feedback? We\\'d be happy to discuss it.": "고객요청? 회의 해 보겠다. ",
   "Read our story, how we started.": "우리 글을 읽어줘 어떻게 시작했을까?!",
-  "Frequently Asked Question": "질의 응답(자주 묻는 질문)",
   "자주 묻는 질문": "자주 묻는 질문 답변",
   "faq": {
+    "title": "질의 응답(자주 묻는 질문)",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/ko.json:62]:
```suggestion
```

File: langDir/nl.json [hunk 12]
```diff
@@ -42,6 +42,9 @@
   "Add Token": "Voeg token toe",
   "Mint": "Mint",
   "faq": {
+    "title": "Veelgestelde vragen",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/nl.json:47]:
```suggestion
```

File: langDir/pl.json [hunk 13]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "Sekcja pytań dla nowych użytkowników",
   "Feedback? We\\'d be happy to discuss it.": "Podziel się swoim komentarzem lub opinią",
   "Read our story, how we started.": "Jak to się wszystko zaczęło...",
-  "Frequently Asked Question": "Najważniejsze Pytania",
   "faq": {
+    "title": "Najważniejsze Pytania",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/pl.json:61]:
```suggestion
```

File: langDir/pt.json [hunk 14]
```diff
@@ -74,10 +74,12 @@
   "Layout Options": "Opções de Layout",
   "Gallery Items Per Page": "Itens da Galeria por Página",
   "Collections Per Page": "Coleções Por Página",
-  "Frequently Asked Question": "Perguntas Frequentemente Feitas",
   "I am helping to cover costs": "Eu estou ajudando a cobrir os custos",
   "I do not want to support": "Eu não quero apoiar",
   "faq": {
+    "title": "Perguntas Frequentemente Feitas",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/pt.json:82]:
```suggestion
```

File: langDir/sk.json [hunk 15]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "Ste tu noví? Máme Často kladené otázky",
   "Feedback? We\\'d be happy to discuss it.": "Máte spätnú väzbu? Radi to prediskutujeme.",
   "Read our story, how we started.": "Prečítajte si náš príbeh, ako sme začali.",
-  "Frequently Asked Question": "Často kladené otázky",
   "faq": {
+    "title": "Často kladené otázky",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/sk.json:61]:
```suggestion
```

File: langDir/tu.json [hunk 16]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "Yeni misiniz? Sıkça sorulan soruları ziyaret ediniz.",
   "Feedback? We\\'d be happy to discuss it.": "Geri dönüs? Görüsünüzü dinlemekten mutluluk duyarız",
   "Read our story, how we started.": "Nasıl başladığımızın hikayesini okuyun",
-  "Frequently Asked Question": "Sıkça sorulan sorular",
   "faq": {
+    "title": "Sıkça sorulan sorular",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/tu.json:61]:
```suggestion
```

File: langDir/ua.json [hunk 17]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "New here? We have F.A.Q.",
   "Feedback? We\\'d be happy to discuss it.": "Feedback? We'd be happy to discuss it.",
   "Read our story, how we started.": "Read our story, how we started.",
-  "Frequently Asked Question": "Frequently Asked Question",
   "faq": {
+    "title": "Frequently Asked Questions",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/ua.json:61]:
```suggestion
```

File: langDir/ur.json [hunk 18]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "ہیں F.A.Q کیا تم یہاں نئے ہو؟ ہمارے پاس",
   "Feedback? We\\'d be happy to discuss it.": "آپ کی رائے؟ ہمیں اس پر تبادلہ خیال کرکے خوشی ہوگی۔",
   "Read our story, how we started.": "ہماری کہانی پڑھیں ، ہم نے کیسے شروعات کی۔",
-  "Frequently Asked Question": "اکثر پوچھے گئے سوال",
   "faq": {
+    "title": "اکثر پوچھے گئے سوال",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/ur.json:61]:
```suggestion
```

File: langDir/vt.json [hunk 19]
```diff
@@ -55,8 +55,10 @@
   "New here? We have F.A.Q.": "Người mới? Chúng tôi có F.A.Q cho bạn",
   "Feedback? We\\'d be happy to discuss it.": "Phản hồi? Chúng tôi rất vui lòng khi thảo luận về nó.",
   "Read our story, how we started.": "Đọc câu chuyện của chúng tôi và cách mà chúng tôi bắt đầu.",
-  "Frequently Asked Question": "Câu hỏi thường gặp",
   "faq": {
+    "title": "Câu hỏi thường gặp",
+    "box_heading1": "",
+    "box_heading2": "",
```
Comment (roiLeo) [inline langDir/vt.json:61]:
```suggestion
```

**Review by roiLeo — COMMENTED** [review]
- remove empty `box_heading` this will automatically fallback to english
- add new translation to page

**Review by roiLeo — APPROVED** [review]

**Pull request merged**